src/github.com/matrix-org/dendrite/clientapi/writers/sendmessage.go

```diff
@@ -3,13 +3,26 @@ package writers
 import (
 	"net/http"
 
+	log "github.com/Sirupsen/logrus"
+	"github.com/matrix-org/dendrite/clientapi/auth"
 	"github.com/matrix-org/util"
 )
 
 // SendMessage implements /rooms/{roomID}/send/{eventType}
 func SendMessage(req *http.Request, roomID, eventType string) (interface{}, *util.HTTPError) {
 	logger := util.GetLogger(req.Context())
-	logger.WithField("roomID", roomID).WithField("eventType", eventType).Info("Doing stuff...")
+	userID, err := auth.VerifyAccessToken(req)
```

Comment:
Should we be doing this here or in `routing.go`? It feels like maybe `routing.go` but I'm unsure.